tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs

```diff
@@ -364,6 +364,58 @@ public void ShowSchedule_PlayoffScheme_RoundNamesAreCreated()
             CollectionAssert.AreEqual(actual.RoundNames, expected.RoundNames);
         }
 
+        /// <summary>
+        /// Test for ShowSchedule method.
+        /// Valid schedule is passed, no exception occurred.
+        /// </summary>
+        [TestMethod]
+        public void ShowSchedule_PlayoffScheme_TeamsNamesAreAssigned()
+        {
+            // Arrange
+            const int TEST_ROUND_COUNT = 6;
+            var tournament = CreateTournamentData(TEST_ROUND_COUNT);
+            tournament.Scheme = TournamentSchemeEnum.PlayOff;
+
+            SetupGetScheduleInfo(
+                TEST_TOURNAMENT_ID,
+                tournament);
+            SetupGetTournamentResults(
+                TEST_TOURNAMENT_ID,
+                new GameServiceTestFixture().TestGamesWithNoNamesForPlayoffRounds(TEST_ROUND_COUNT).Build());
+            
+            var expectedTeamsNames = new List<string>
+            {
+                // Every game in first round
+                "Team 1",
+                "Team 2",
+                // Bronze game
+                "Looser61",
+                "Looser62",
+                // Final game
+                "Winner61",
```

Comment:
it is not clear from the test why 61 is expected

Reply:
Ok. changing back.